### PR TITLE
Add methods and tests to calculate highest and lowest score, biggest …

### PR DESCRIPTION
**Pull request by foleymichelle9**
Added methods and tests for highest/lowest total score, biggest blowout, and percenatge of visitor and home wins

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -41,4 +41,16 @@ def initialize(game_parameter)
     @venue = game_parameter[:venue]
     @venue_link = game_parameter[:venue_link]
   end
+
+  def highest_total_score
+    @@all.map { |game| game.away_goals + game.home_goals}.max
+  end
+
```
Comment (chrisstankus99) [inline lib/game.rb:48]: Awesome job on these methods!

File: lib/team.rb [hunk 2]
```diff
@@ -16,6 +16,7 @@ def self.create_teams(csv_file_path)
 
   def self.all
     @@all
+
```
Comment (margoflewelling) [inline lib/team.rb:19]: Could probably delete this line, looks accidental

File: lib/game_team.rb [hunk 3]
```diff
@@ -51,4 +51,25 @@ def initialize(game_team_parameter)
...
   end
+
+  def percentage_home_wins
+    all_home_games = @@all.find_all do |game_team|
+      game_team.hoa == "home"
+    end
+    home_wins = all_home_games.find_all do |game_team|
+       game_team.result == "WIN"
+    end
+    ((home_wins.length / all_home_games.length.to_f) * 100).round(2)
```
Comment (chrisstankus99) [inline lib/game_team.rb:62]: We could probably refactor this into a module or helper method if we have time.

File: lib/game.rb [hunk 4]
```diff
@@ -41,4 +41,16 @@ def initialize(game_parameter)
...
+  def lowest_total_score
+    @@all.map { |game| game.away_goals + game.home_goals}.min
+  end
+
+  def biggest_blowout
+    @@all.map { |game| (game.away_goals - game.home_goals).abs}.max
```
Comment (margoflewelling) [inline lib/game.rb:54]: Good logic on this method!

**Pull request merged**